src/utils/enums.ts

```diff
@@ -0,0 +1,4 @@
+export enum AccountDropdown {
+  Version1 = 1,
+  Version2 = 2,
```

Comment:
`Version1` is a vague naming. It's unclear what do they do and for what they are responsible.
`Version2` is not used anywhere

Reply:
`Version2` will be used later in the next PR.